x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/__mocks__/roles.ts

```diff
@@ -5,19 +5,21 @@
  * 2.0.
  */
 
+import { engines } from '../../../app_search/__mocks__/engines.mock';
+
 import { AttributeName } from '../../types';
 
 export const asRoleMapping = {
-  id: null,
+  id: 'sdgfasdgadf123',
   attributeName: 'role' as AttributeName,
-  attributeValue: ['superuser'],
+  attributeValue: 'superuser',
   authProvider: ['*'],
   roleType: 'owner',
   rules: {
     role: 'superuser',
   },
   accessAllEngines: true,
-  engines: [],
+  engines,
```

Comment:
Does it feel at all weird to have an App-Search-only `engines` prop in a shared mock? No worries if not of course, just checking/thinking out loud

Reply:
It is used in a [shared component](https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/role_mappings_table.test.tsx#L14) to test both cases